pkg/audit/verifier.go

```diff
@@ -93,9 +93,10 @@ func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe, skip map[s
 		return nil, err
 	}
 
+	// note: offlineNodes here will include disqualified nodes
```

Comment:
Hmm... this means that DQ nods will appear as offline in the report. DQ nodes should not appear in the report at all.

Reply:
this should probably be better handled but Moby and I had the justification that when UpdateStats is called when reporting, the disqualified nodes will be ignored/their stats won't be updated.

Reply:
alternately, CreateAuditOrderLimits could also a return a list of nodes that were disqualified, and those could be skipped but ignored by the report.

Reply:
I think you made the right call for now. We need to think about it. Because it won't be DQ nodes only. Soon we will have nodes that exited gracefully too.